### PR TITLE
bump plenum version and update plenum code

### PR DESCRIPTION
**Pull request by lovesh**
Signed-off-by: lovesh harchandani <lovesh.bond@gmail.com>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -56,7 +56,7 @@
     data_files=[(
         (BASE_DIR, ['data/nssm_original.exe'])
     )],
-    install_requires=['indy-plenum-dev==1.4.419',
+    install_requires=['indy-plenum-dev==1.4.420g',
```
Comment (ashcherbakov) [inline setup.py]: A typo `g`?

Comment (lovesh) [reply setup.py]: Yes, i realised but was waiting for another release. Thanks for catching that

**Comment by jerrysen** [comment]
please test this

**Comment by jerrysen** [comment]
test this please

**Pull request merged**